alwaysontop/render.js

```diff
@@ -319,9 +319,17 @@ class AlwaysOnTop extends EventEmitter {
             this._alwaysOnTopWindow.document.body.innerHTML = `
               <div id="react"></div>
               <video autoplay="" id="video" style="transform: none;" muted></video>
+              <div class="dismiss">&times;</div>
               <link rel="stylesheet" href="file://${ cssPath }">
             `;
 
+            this._alwaysOnTopWindow.document.body.addEventListener('click', e => {
```

Comment:
isn't it better to attach the listener on the element instead of the body?

Reply:
in this case this is more explicit, because we are setting innerHTML

Reply:
ok but performance-wise, isn't it going to be better to put the lister on the element?

Also I'm wondering if alwaysontop.js is better place for this code. WDYT?

Reply:
doesn't matter tbh but found a workaround